### PR TITLE
Admin messaging - Auction show page - Reject work

### LINKED ISSUE #1015: Admin messaging - Auction show page - Reject work
As an admin, I want a way to easily mark an auction as rejected, without having to go into the edit form. So I should be able to do this on the show page of an auction

- [x] **Scenario: Admin marks an auction as rejected from the show page**
- Given I am an authenticated admin
- And a vendor has delivered code for an auction (clicked the "I'm Done" button)
- When I view the show page for that auction
- Then I should see the following messaging in the status box

> Pending acceptance
> [auction.winner] made a pull request and marked this auction as having been delivered. If their work meets this auction’s acceptance criteria, click “accept.” Otherwise, click “reject.”

- And I should see an "Accept" and a "Reject" button
- And "[Auction.Winner]" should link to the winning vendor's profile (https://micropurchase.18f.gov/admin/users/[id])
- And "pull request" should link to the Delivery URL for that auction

screen shot 2016-08-08 at 6 45 48 am

- [x] **Scenario: Admin clicks "reject" button on auction show page**
- Given I am an authenticated admin
- And I am on the show page for an auction that has work delivered (the vendor has clicked "I'm Done")
- When I click to the "Reject" button
- Then the auction should be marked as rejected by me
- And I should see the following messaging in the status box

> Rejected
> [Vendor.name]'s work was rejected on [auction.rejectionDate].

- And "[Vendor.name]'s work" should link to the delivery URL

## Notes

- Status box appears at the top right of the auction show page

screen shot 2016-08-08 at 6 59 48 am

### PR DESCRIPTION
**Pull request by jessieay**
* Closes #1015 

When pending acceptance:
screen shot 2016-08-23 at 3 58 19 pm

After clicking "reject":

screen shot 2016-08-23 at 4 00 46 pm

### CODE REVIEW

File: app/views/admin/auctions/_accept_or_reject.html.erb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<%= link_to I18n.t('statuses.c2_presenter.pending_acceptance.actions.accept'),
+  '#',
+  class: 'usa-button usa-button-outline action-button' %>
+<%= link_to I18n.t('statuses.c2_presenter.pending_acceptance.actions.reject'),
+  admin_auction_rejection_path(status.auction),
+  method: :patch,
```
Comment (harrisj) [inline app/views/admin/auctions/_accept_or_reject.html.erb]: Ha, the patch method

**Comment by harrisj** [comment]
Looks great, but it need a rebase. Sorry

**Comment by jessieay** [comment]
@harrisj rebased!

**Comment by harrisj** [comment]
LGTM. Merging!

**Pull request merged**